broker/Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'systemu'
 gem 'mongoid'
 gem 'bson'
 gem 'bson_ext'
+gem 'net-ssh'
```

Comment:
Looks like ruby193-rubygem-net-ssh-2.7.0 is in the repos for origin and ose but it's not currently installed via any dependency so it'd need to be added to the specfile as a Requires

    Requires:      %{?scl:%scl_prefix}rubygem-net-ssh